task/validate-api/src/lib.rs

```diff
@@ -45,6 +47,12 @@ pub enum ValidateOk {
     Removed = 3,
 }
 
+#[derive(Copy, Clone, Debug, SerializedSize, Serialize, Deserialize)]
+pub struct MuxSegment {
+    pub mux: drv_i2c_api::Mux,
+    pub segment: drv_i2c_api::Segment,
```

Comment:
Not strictly necessary since all these crates live in our workspace, but I'd publicly re-export these types so a client of `validate-api` doesn't have to also explicitly depend on `drv-i2c-api` to dig into a `MuxSegment`.